### PR TITLE
Some tests emphasizing ill-conditioning of calculations involving the eccentricity.

### PR DESCRIPTION
**Pull request by eggrobin**
These are mostly due to naïvely computing 1-e²; other tests will be needed for specifications of the conic parameters that do not include e.
Part of #967.

### CODE REVIEW

File: physics/kepler_orbit_test.cpp [hunk 1]
```diff
@@ -170,6 +170,50 @@ class KeplerOrbitTest : public ::testing::Test {
     return elements;
   }
 
+  // An ellipse with a = 1 au, e very close to 1.
+  static KeplerianElements<ICRFJ2000Equator> NearlyParabolicEllipse() {
+    KeplerianElements<ICRFJ2000Equator> elements;
+    // This is about half the precision at 1, which maximizes the error in the
```
Comment (pleroy) [inline physics/kepler_orbit_test.cpp]: "half the precision" is unclear.  It's in number of bits.

File: physics/kepler_orbit_test.cpp [hunk 2]
```diff
@@ -170,6 +170,50 @@ class KeplerOrbitTest : public ::testing::Test {
...
+    // naïve evaluation of 1 - (1 - ε)².  Note the 1 - (1 - x), to ensure that
+    // (1 - ε) is exact.
+    // Note that [1 - (1 - ε)²] = [[2ε] - [ε²]], where [] denotes rounding.
+    constexpr double ε = 1 - (1 - 2e-9);
+    constexpr double ε² = ε * ε;
+    constexpr double ε³ = ε * ε * ε;
```
Comment (pleroy) [inline physics/kepler_orbit_test.cpp]: Use `ε²`.

Comment (eggrobin) [inline physics/kepler_orbit_test.cpp]: Done.

File: physics/kepler_orbit_test.cpp (hunk 1 above)
Comment (eggrobin) [inline physics/kepler_orbit_test.cpp]: Done.

**Pull request merged**